bootstrap/data_ingester.py

```diff
@@ -0,0 +1,139 @@
+'''
+ * ADOBE CONFIDENTIAL
+ * ___________________
+ *
+ *  Copyright 2018 Adobe Systems Incorporated
+ *  All Rights Reserved.
+ *
+ * NOTICE:  All information contained herein is, and remains
+ * the property of Adobe Systems Incorporated and its suppliers,
+ * if any.  The intellectual and technical concepts contained
+ * herein are proprietary to Adobe Systems Incorporated and its
+ * suppliers and are protected by all applicable intellectual property
+ * laws, including trade secret and copyright laws.
+ * Dissemination of this information or reproduction of this material
+ * is strictly forbidden unless prior written permission is obtained
+ * from Adobe Systems Incorporated.
+'''
+
+import json
+from helper import setup_logger, http_request
+
+
+LOGGER = setup_logger(__name__)
+FILE_PATH = "../datasets/retail/XDM0.9.9.9/"
+CONTENT_TYPE = "application/json"
+
+
+def get_dataset_id(create_dataset_url, headers, dataset_title, schema_id):
+    """
+    Get the datasetId by making a POST Request to "/data/foundation/catalog/datasets?requestDataSource=true"
+    :param create_dataset_url: url
+    :param headers: headers
+    :param dataset_title: dataset title
+    :param schema_id: schema url
```

Comment:
Should this be, `schema id`?

Reply:
This has to be schema because the schema, mixin and class are ids but the values are the urls unlike dataset or batch id.